mass_editing/models/mass_editing_line.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+# Copyright (C) 2020 Iván Todorovich (https://twitter.com/ivantodorovich)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class MassEditingLine(models.Model):
+    _name = "mass.editing.line"
+    _description = "Mass Editing Line"
+    _order = "sequence, field_id"
+
+    sequence = fields.Integer()
+    server_action_id = fields.Many2one(
+        "ir.actions.server",
+        string="Server Action",
+        ondelete="cascade",
+        required=True,
+    )
+    model_id = fields.Many2one(
+        "ir.model",
+        related="server_action_id.model_id",
+    )
+    field_id = fields.Many2one(
+        "ir.model.fields",
+        string="Field",
+        domain="""
+            [
+                ("name", "!=", "__last_update"),
+                ("ttype", "not in", ["reference", "function"]),
+                ("model_id", "=", model_id),
+            ]
+        """,
+        ondelete="cascade",
+        required=True,
+    )
+    widget_option = fields.Char(
+        string="Widget Option",
+        help="Add widget text that will be used to display the field in the wizard.\n"
+        "Example: 'many2many_tags', 'selection', 'image'",
+        compute="_compute_widget_option",
+        readonly=False,
+        store=True,
+    )
+    apply_domain = fields.Boolean(
```

Comment:
Hi. Could you say the reason of this field. I dont understand why it exists. It is new ?

Reply:
Thanks for the review @legalsylvain  :)

This was introduced by @victoralmau here: https://github.com/OCA/server-ux/pull/237
> Add apply domain option in lines to fix some many2one fields with default domain (company_id for example)

Reply:
@legalsylvain this was for fixing something you introduce with your refactoring, and it's that if you apply the default field's domain, you need to add the fields used in the domain to the view, or you will get an error, so the apply domain is now optional. If applied, you need to add the needed fields. Clear example in v13: fields with `check_company=True`. They need to add always `company_id` field in the view, or you will get an error.